packages/inscription-protocol/src/data/inscription.ts

```diff
@@ -61,7 +66,7 @@ export interface Inscription {
   restEntityInfoRequest: RestEntityInfoRequest;
   restResource: RestResource;
   restResourceRequest: RestResourceRequest;
-  roleMeta: RoleMeta[];
+  roleMeta: RoleMeta;
```

Comment:
@ivy-rew or @ivy-lli anybody know why this has changed? I didn't changed here anything:
https://github.com/axonivy/core/pull/8262/files

Reply:
this is not stable, but also not important. 
The inscription type is only here that all this types inside are created, e.g. RoleMeta. If it is an array or not makes no difference.

Reply:
so I keep it here as it is. because it was generated with the command
`npm run protocol:inscription generate`